mobile-widgets/qml/CloudCredentials.qml

```diff
@@ -140,7 +139,8 @@ Item {
 				onClicked: {
 					manager.setGitLocalOnly(true)
 					PrefCloudStorage.cloud_auto_sync = false
-					prefs.credentialStatus = CloudStatus.CS_NOCLOUD
+					prefs.oldStatus = PrefCloudStorage.cloud_verification_status
```

Comment:
So this is net new code that doesn't appear to be explained in the commit message. Why are you saving the `oldStatus` here?

Reply:
or is this the odd reference to `does not hit the "if"` - which I have no idea what that tells me...

Reply:
explained (hopefully in an understandable way).